webextensions/api/tabs.json

```diff
@@ -2740,7 +2737,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/tabs/onReplaced",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "26"
```

Comment:
✅ https://github.com/chromium/chromium/commit/a28526c12d8283c7c4fa7f43963664746a839c59 in [26.0.1377.0](https://github.com/chromium/chromium/releases/tag/26.0.1377.0) before 26.0.1410.